spring-ai-model/src/main/java/org/springframework/ai/model/tool/DefaultToolCallingChatOptions.java

```diff
@@ -198,6 +201,16 @@ public void setTopP(@Nullable Double topP) {
 		this.topP = topP;
 	}
 
+	@Override
+	@Nullable
+	public Boolean getThink() {
+		return this.think;
```

Comment:
As with ChatOptions, the boolean itself isn't sufficient across all model providers.